spec/chatbox.js

```diff
@@ -1224,7 +1224,8 @@
                         // the page is loaded via file:///
                         return;
                     }
-                    var message = document.URL.split(window.location.pathname)[0] + "/logo/conversejs.svg";
+                    var message = document.URL.split(window.location.pathname)[0] + "/logo/conversejs.svg?param1=val1&param2=val2",
+                        htmlEscapedMessage = message.replace(/&/g, '&amp;');
```

Comment:
Instead of modifying the existing check, please add an additional check for the above URL, so that both the original and the new URL are checked.